### DISCUSSION TITLE
Not able to add the config inside src folder

### DISCUSSION THREAD

**Discussion by acivitillo** [sveltia/sveltia-cms#280] [Q&A] [answered]
So, I have the below situation that works:

- `config.yaml` inside an `admin` folder in the root
- `index.html` inside an `admin` folder in `root/src`

You can also see the below screenshot. This works, but when a user uploads an image, it saves images in `root/images`. I tried to fix it, but then eleventy appends `src` so I end up with `src/src/images`. Is there a way to make sveltia read the `config.yaml` inside `src/admin` and to make it treat `src` as the root?

![Uploading image.png…]()

**Comment by kyoshino** [comment, marked as answer]
This is more like a 11ty question 😆 

I see `/src/admin/config.yml` is not copied to `/_site/admin` for some reason, probably because YAML files are special. And actually, the location of the config file doesn’t affect the root, which is always the project root.

Here’s what I’ve done: https://github.com/dyvenia/site-thescalableway/compare/main...kyoshino:site-thescalableway:sveltia-280

You should be able to see the images both in the CMS and on the site 👍🏼
